YadavAkhileshh/Alien-Invasion-Defense
# Updated Alignments of buttons and improved layout

## What does this PR do?

This PR fixes the alignment and overlapping of elements like theme changing switch, music volume controller, sign-up and player instructions buttons, Start button, feedback button, Difficulty level slector button and displays like live-life, live-score, high-score and game-info. The animated snowflakes background was replaced with solid dark blue background. The duplicates of start button, game-info display, theme changing switch and music volume controller were removed from the earlier UI. All the unnecessary duplicates of the elements present inside the game container were removed, along with blurred background. The "Orbitron" font has been applied to the entire UI. The function of Start button was also fixed, where earlier it was not working and the game was not displaying when clicked. The colours of sign-up button, player instructions button, Start button were enhanced a bit with some hover effect.


Fixes #444 (UI/UX Improvements: Overlapping Elements, Readability, and Layout Issues)

<img width="958" alt="game2" src="https://github.com/user-attachments/assets/3b04c028-d10c-4249-a9fd-d7148ce2e365"> [OLDER UI]

<img width="960" alt="2024-10-25 (2)" src="https://github.com/user-attachments/assets/5b759f15-9df6-45d7-97ff-ae6876bc5eb5">  [NEW UI]


## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Chore (refactoring code, technical debt, workflow improvements)


## How should this be tested?



1. Test A: Start Button Fix
   - Ensure that the game starts properly when the "Start Game" button is clicked.
   - Verify that the game interface displays correctly without errors.
   - Open the game interface in the browser.
   - Click on the "Start Game" button.
   - Confirm that the game starts and no errors appear in the console.
2. Test B: UI layout & alignment
   - Verify that all the elements, buttons and display elements are correctly aligned across desktop view.
   - Open the game in a desktop view.
   - Verify that all UI elements are aligned vertically in the right-hand section (music volume, light mode, instructions, signup).
   - Confirm that the feedback button is positioned correctly at the bottom-right corner.

## Mandatory Tasks
- Self-reviewed the code and removed unnecessary codes.
- Verified the working of "Start Game" button.
- Checked alignments of UI elements.
